### PR TITLE
Containerize the Queuer Service

### PR DESCRIPTION
**Pull request by Emmanuel289**
- [ ] I have added the tests to cover my changes.
- [x] I have updated the documentation, VERSION, and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

**Review by scottwn — DISMISSED** [review]

File: refactor/queuer/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+pydantic==1.9.0
+fastapi==0.74.0
+nats-py==2.0.0
+python-multipart==0.0.5
+aiolimiter==1.0.0
+fastapi-socketio==0.0.9
+websockets==10.2
+requests==2.24.0
+mock==4.0.3
+pytest==7.1.1
```
Comment (wjcunningham7) [inline refactor/queuer/requirements.txt:12]: things like mock and pytest should remain in the developer requirements file

Comment (Emmanuel289) [reply refactor/queuer/requirements.txt]: Thanks. I have separated the developer requirements from distribution requirements in the `requirements` dir

**Review by scottwn — DISMISSED** [review]

File: refactor/queuer/requirements.txt (hunk 1 above)
Comment (scottwn) [reply refactor/queuer/requirements.txt]: @wjcunningham7 @Emmanuel289 I can merge this if it's ready to go

Comment (Emmanuel289) [reply refactor/queuer/requirements.txt]: Yes please, it's ready to go @wjcunningham7 @scottwn.

Comment (Emmanuel289) [reply refactor/queuer/requirements.txt]: Had to update changelog and version to latest release.

**Review by scottwn — APPROVED** [review]

**Pull request merged**